frontend/public/components/deploy-image.tsx

```diff
@@ -39,6 +39,13 @@ export class DeployImage extends React.Component<DeployImageProps, DeployImageSt
     };
   }
 
+  static getDerivedStateFromProps(props: DeployImageProps, state: DeployImageState) {
+    const params = new URLSearchParams(props.location.search);
+    const ns = params.get('ns');
+
+    return {namespace: ns};
```

Comment:
This seems like it should go in the constructor rather than `getDerivedStateFromProps`

Reply:
Fixed.  Thanks!